app/views/home/portfolio.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1 class="text-center">Portfolio page mother fuckets!</h1>
+
+<table class="table">
+  <thead>
+    <tr>
+      <th scope="col">broker #</th>
+      <th scope="col">name</th>
+      <th scope="col">symbol</th>
+      <th scope="col">latest price</th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% current_broker.stocks.each do |stock| %>
+      <tr>
+        <th scope="row"><%= current_broker.id %></th>
+        <td><%= stock.name %></td>
+        <td><%= stock.symbol %></td>
+        <td><%= stock.current_price %></td>
+        <td><%= button_to "remove stock", stock_path(stock), method: :delete %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
```

Comment:
add new line at the end of file

Reply:
rubocop -a removes those spaces in the EOL.
but rubocop also adds the space in the EOL itself it turn gives errors in the workflow.

that's why we always remove that new line after the EOL